network/wifi/at_parser.h

```diff
@@ -299,11 +312,21 @@ union in_out_param {
  * @brief Parameter to initialize parser
  */
 struct at_init_param {
-	/* Should be initialized outside in order to fil uart_irq_conf */
+	/* Should be initialized outside in order to fill uart_irq_conf */
 	struct uart_desc	*uart_desc;
 	struct irq_ctrl_desc	*irq_desc;
 	uint32_t		uart_irq_id;
 	void			*uart_irq_conf;
+	/* Will be called when a new connection is created or deleted.
+	When an AT_NEW_CONNECTION event is received, user can save in cb a
+	circular buffer where to write data received from the connection.
+	If *cb is set to NULL data will not be saved */
+	void			(*connection_callback)(void *ctx,
```

Comment:
remove extra tabs between return type and function pointer.

Reply:
I used the tab to indent with the other fields